### ISSUE TITLE
--load-extension=spatialite not working with datasetteproject/datasette docker image

### ISSUE THREAD

**Issue by danp** [simonw/datasette#1114]
https://github.com/simonw/datasette/commit/6aa5886379dd9017215904fb28567b80018902f9 added the `--load-extension=spatialite` shortcut looking for the extension in these places:

https://github.com/simonw/datasette/blob/12877d7a48e2aa28bb5e780f929a218f7265d849/datasette/utils/__init__.py#L56-L60

However, in the datasetteproject/datasette docker image the file is at `/usr/local/lib/mod_spatialite.so`.

This results in the example command [here](https://docs.datasette.io/en/stable/installation.html#loading-spatialite) failing:

```
% docker run --rm -p 8001:8001 -v `pwd`:/mnt datasetteproject/datasette datasette -p 8001 -h 0.0.0.0 /mnt/data.db --load-extension=spatialite
Error: Could not find SpatiaLite extension
```

But it does work when given an explicit path:

```
% docker run --rm -p 8001:8001 -v `pwd`:/mnt datasetteproject/datasette datasette -p 8001 -h 0.0.0.0 /mnt/data.db --load-extension=/usr/local/lib/mod_spatialite.so
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8001 (Press CTRL+C to quit)
...
```

Perhaps `SPATIALITE_PATHS` should include `/usr/local/lib/mod_spatialite.so`?

**Comment by simonw** [comment]
This is a great catch, thanks.

**Comment by danp** [comment]
Thank you!

**Comment by simonw** [comment]
Fix is out in 0.52.1: https://docs.datasette.io/en/latest/changelog.html#v0-52-1

**Comment by simonw** [comment]
The new Docker container is pushed to Docker Hub too. Here's it in action:

```
shapefile-to-sqlite % docker run --rm -p 8001:8001 -v `pwd`:/mnt datasetteproject/datasette datasette --version
datasette, version 0.52.1
shapefile-to-sqlite % docker run --rm -p 8001:8001 -v `pwd`:/mnt datasetteproject/datasette datasette -p 8001 -h 0.0.0.0 /mnt/nps-spatialite.db --load-extension=spatialite
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8001 (Press CTRL+C to quit)
```
